docker/startup_scripts/load_custom_fields.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.contenttypes.models import ContentType
+from extras.models import CF_TYPE_TEXT, CustomField
+
+from dcim.models import Device
+from dcim.models import DeviceType
+
+device      = ContentType.objects.get_for_model(Device)
+device_type = ContentType.objects.get_for_model(DeviceType)
+
+nine_urn, created = CustomField.objects.get_or_create(
+    type=CF_TYPE_TEXT,
+    name='nine_urn',
+    description='Nine URN'
+)
+
+nine_urn.obj_type.add(device)
+nine_urn.obj_type.add(device_type)
```

Comment:
This file seems to contain internal code, but it would suit as a good example as part of the `README.md`.

Reply:
👍